komodo/release_transpiler.py

```diff
@@ -204,20 +204,31 @@ def main():
     matrix_parser.add_argument(
         "--release-folder",
         required=True,
+        type=dir_path,
         help="Folder with existing release file (default: None)",
     )
     matrix_parser.add_argument(
         "--override-mapping",
         required=True,
+        type=check_if_valid_file,
         help="File containing explicit matrix packages (default: None)",
     )
+
+    def checkValidPythonVersion(python_versions: str) -> list:
```

Comment:
We can be more explicit in the return type: `list` can be changed to `List[str]`